src/parser/shared/modules/StatTracker.js

```diff
@@ -28,322 +28,14 @@ class StatTracker extends Analyzer {
-    },
-    268616: { // Swell of Voodoo
-      itemId: 159763, // Idol of Vol'jamba
-      mastery: (_, item) => calculateSecondaryStatDefault(172, 114, item.itemLevel),
-    },
-    273988: { // Primal Instinct
-      itemId: 158155, // Dinobone Charm
-      strength: (_, item) => calculatePrimaryStat(280, 351, item.itemLevel),
-      agility: (_, item) => calculatePrimaryStat(280, 351, item.itemLevel),
-      intellect: (_, item) => calculatePrimaryStat(280, 351, item.itemLevel),
-    },
-    269885: { // Residual Viciousness
-      itemId: 159977, // Vindictive Golem Core
-      crit: (_, item) => calculateSecondaryStatDefault(172, 170, item.itemLevel),
-    },
-    273992: { // Speed of the Spirits
-      itemId: 158154, // Zandalari Bijou
-      haste: (_, item) => calculateSecondaryStatDefault(280, 414, item.itemLevel),
-    },
-    268604: { // Blood Crazed
-      itemId: 159075, // Bloodhex Talisman
-      crit: (_, item) => calculateSecondaryStatDefault(172, 207, item.itemLevel),
-    },
-    271103: { // Rezan's Gleaming Eye
-      itemId: 158712, // Rezan's Gleaming Eye
-      haste: (_, item) => calculateSecondaryStatDefault(300, 455, item.itemLevel),
-    },
-    268836: { // Blood of My Enemies
-      itemId: 159625, // Vial of Animated Blood
-      strength: (_, item) => calculatePrimaryStat(300, 705, item.itemLevel),
-    },
-
-    // endregion
-    // region World boss
-    278227: { // Barkspines
-      itemId: 161411, // T'zane's Barkspines active TODO: Make an analyzer
-      crit: (_, item) => calculateSecondaryStatDefault(355, 1160, item.itemLevel), // TODO: Verify stats and if it scales with this formula (might be trinket/jewerly scaling)
-    },
-    // endregion
-    // region Dungeons
-    [SPELLS.CONCH_OF_DARK_WHISPERS_BUFF.id]: { // Conch of Dark Whispers
-      itemId: ITEMS.CONCH_OF_DARK_WHISPERS.id,
-      crit: (_, item) => calculateSecondaryStatDefault(300, 455, item.itemLevel),
-    },
-    271115: { // Ignition Mage's Fuse
-      itemId: ITEMS.IGNITION_MAGES_FUSE.id,
-      haste: (_, item) => calculateSecondaryStatDefault(310, 233, item.itemLevel),
-    },
-    [SPELLS.KINDLED_SOUL.id]: { // Balefire Branch trinket's buff (stack starts at 100)
-      itemId: ITEMS.BALEFIRE_BRANCH.id,
-      intellect: (_, item) => calculatePrimaryStat(340, 12, item.itemLevel),
-    },
-    [SPELLS.BENEFICIAL_VIBRATIONS.id]: {
-      itemId: ITEMS.AZEROKKS_RESONATING_HEART.id,
-      agility: (_, item) => calculatePrimaryStat(300, 593, item.itemLevel),
-    },
-    // endregion
-    // endregion
-
-    // region Racials
-    // Mag'har Orc
-    [SPELLS.RICTUS_OF_THE_LAUGHING_SKULL.id]: { crit: 411 },
-    [SPELLS.ZEAL_OF_THE_BURNING_BLADE.id]: { haste: 411 },
-    [SPELLS.FEROCITY_OF_THE_FROSTWOLF.id]: { mastery: 411 },
-    [SPELLS.MIGHT_OF_THE_BLACKROCK.id]: { versatility: 411 },
-    // endregion
-  };
-
   statBuffs = {};
 
   _pullStats = {};
   _currentStats = {};
 
-  statMultiplier = {
```

Comment:
`statMultiplier` is an internal object for `StatTracker`, not a buff list. It should stay in StatTracker. the other stuff seems fine.

Reply:
@anontyro just checking in. are you planning to follow up on this PR?